modules/shortcodes/slideshare.php

```diff
@@ -91,14 +91,16 @@ function slideshare_shortcode( $atts ) {
 		$player .= " frameborder='" . intval( $attr['fb'] ) . "'";
 	}
 
-	// check the margin width; if not empty, cast as int.
-	if ( ! empty( $attr['mw'] ) || '0' === $attr['mw'] ) {
-		$player .= " marginwidth='" . intval( $attr['mw'] ) . "'";
-	}
+	if ( class_exists( 'Jetpack_AMP_Support' ) && ! Jetpack_AMP_Support::is_amp_request() ) {
```

Comment:
@carlosenamdev If the class doesn't exist, we should execute in the statement as the render mode will very unlikely be AMP. This should be `if ( ! class_exists(...) || ! Jetpack_Amp... )`

Reply:
@MattGeri Thanks for noticing that, it's fixed in the last commit.